streams/service/src/main/java/com/hortonworks/streamline/streams/service/SchemaResource.java

```diff
@@ -132,4 +133,61 @@ public Response getKafkaSourceSchema(@PathParam("topicName") String topicName,
         }
     }
 
+    @GET
+    @Path("/{schemaName}/versions")
+    @Timed
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAllSchemaVersions(@PathParam("schemaName") String schemaName,
+                                         @Context SecurityContext securityContext) {
+        try {
+            LOG.info("Get all versions for schema : {}", schemaName);
+            Collection<SchemaVersionInfo> schemaVersionInfos = schemaRegistryClient.getAllVersions(schemaName);
+            Collection<String> schemaVersions = new ArrayList<>();
```

Comment:
@satishd Because of the flux translation layer its easiest to make everything a string. Thats the approach I have taken for all kafka consumer and producer related properties - to keep them as string and handle the conversion where needed.